ee/tabby-webserver/src/schema/auth.rs

```diff
@@ -244,6 +244,7 @@ pub struct User {
     pub created_at: DateTime<Utc>,
     pub active: bool,
     pub is_password_set: bool,
+    pub avatar: Option<String>,
```

Comment:
```suggestion
```

Add http api interface (accessible by any login user) `/avatar/{user_id}` to return avatar image